wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -119,6 +119,12 @@
    */
   public static final String WORKSPACE_TOOLING_PLUGINS_ATTRIBUTE = "plugins";
 
+  /**
+   * Template for workspace attribute key that sets sidecar limit in a plugin.
+   * %s should be replaced with plugin ID. When plugin provides several sidecars this property sets
+   * the same limit for each sidecar, so is not that useful in such a case.
+   * Value format see {@link KubernetesSize}
```

Comment:
Well, format is described on it class java doc but in `toBytes` method java docs. Consider moving format description to class java docs or reference `KubernetesSize#toBytes`  here